lib/elixir/lib/enum.ex

```diff
@@ -3671,6 +3695,12 @@ defmodule Enum do
     end
   end
 
+  # TODO: Remove me on v2.0
+  defp aggregate(%{__struct__: Range, first: first, last: last} = range, fun, empty) do
```

Comment:
I believe we don't need this clause because it will fallback to the protocol.

Reply:
You're right
```
iex(9)> Enum.max(old_range)                                                 
10
```